var/spack/repos/builtin/packages/py-pynrrd/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPynrrd(PythonPackage):
+    """Python library for reading and writing NRRD files into and from numpy arrays"""
+
+    homepage = "https://github.com/mhe/pynrrd"
+    url = "https://pypi.io/packages/source/p/pynrrd/pynrrd-0.3.5.tar.gz"
```

Comment:
doesn't matter but we have been specifying tarballs from GitHub directly : https://github.com/mhe/pynrrd/releases